src/assets/icons/likeRed.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const LikeRed = () => {
+  return (
+    <svg 
+      id="Layer_1" 
+      data-name="Layer 1" 
+      xmlns="http://www.w3.org/2000/svg" 
+      viewBox="0 0 122.88 107.39"
+      width="40px"
+      height="40px"
+    >
+      <defs>
+ 
+        <style>
+          {`.cls-1 {
```

Comment:
Can you try and add this style to the `Like` component? If all this component is doing is changing the colour - then it would be better if that was just a prop/functionality on the `Like` component. So we don't have duplicated icons.

Reply:
Okay, no problem!